osu.Game/Overlays/Profile/Header/Components/DailyChallengeStatsTooltip.cs

```diff
@@ -112,16 +112,18 @@ public void SetContent(DailyChallengeTooltipData content)
             background.Colour = colourProvider.Background4;
             topBackground.Colour = colourProvider.Background5;
 
-            currentDaily.Value = UsersStrings.ShowDailyChallengeUnitDay(content.Statistics.DailyStreakCurrent.ToLocalisableString(@"N0"));
+            // todo: ideally we want to use UsersStrings.ShowDailyChallengeUnit{Day,Week}(...), but it's broken right now.
+            // see: https://github.com/ppy/osu/issues/29355#issuecomment-2277139889
+            currentDaily.Value = LocalisableString.Interpolate($"{statistics.DailyStreakCurrent.ToLocalisableString("N0")}d");
             currentDaily.ValueColour = colours.ForRankingTier(TierForDaily(statistics.DailyStreakCurrent));
 
-            currentWeekly.Value = UsersStrings.ShowDailyChallengeUnitWeek(statistics.WeeklyStreakCurrent.ToLocalisableString(@"N0"));
+            currentWeekly.Value = LocalisableString.Interpolate($"{statistics.WeeklyStreakCurrent.ToLocalisableString(@"N0")}w");
```

Comment:
We should add concrete localisations for these instead imo. Duplicate the osu!web ones for our purpose.

Reply:
Feel free to apply that.